docs/facts/known_issues.rst

```diff
@@ -30,3 +30,26 @@ However, the right way to do this is by first locate where your's
 
 :code:`find /usr/lib/ -type f -iname 'libffi.so.*'` and then apply the
 softlink to :code:`libffi.so.7`
+
+Urllib3
```

Comment:
I'd likely put this section on hold as (and sorry for misleading this, I don't know where my mind was!) the `pyfunceble-not-resolved` domain is a PyFunceble reserved/generated host which is used when the IP was not found while trying to resolve.

Reply:
Removed by <https://github.com/funilrys/PyFunceble/pull/122/files#diff-5c443d7148beb3d2f8a2ccb24c6a7607>